compile.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -x
+PASS_FILE="magen_keystore.pass"
+KEYSTORE_FILE="magen.pfx"
+IMAGE_TAG_FILE="image.tag"
+ALIAS="magen"
+
+GUES_PATH=/magen/hamagen-react-native
```

Comment:
what does `GUES` stand for?

Reply:
typo from GUEST :)

Reply:
Notify me once you push and I'll resolve